CoreFoundation/String.subproj/CFString.c

```diff
@@ -1264,12 +1264,15 @@ CF_PRIVATE CFStringRef __CFStringCreateImmutableFunnel3(
 	contentsDeallocator = __CFGetDefaultAllocator();
     }
 
+    #ifndef DEPLOYMENT_TARGET_ANDROID
+    // crashes String.swift, line 51 when creating String from zero length Data
```

Comment:
It'd be great to file a https://bugs.swift.org report to track this crash, and include the report number here. That way, we'll know to remove this `#ifdef` once the report is fixed.